test/image/compare_pixels_test.js

```diff
@@ -106,8 +106,10 @@ else {
  *
  */
 function untestableFilter(mockName) {
-    var cond = !(
+    var cond =
+    !(
         mockName === 'font-wishlist' ||
+        mockName.indexOf('gl3d_volume_') !== -1 ||
```

Comment:
Hmm. That's less than ideal. Do you have an idea why we can't pixel-compare `volume` mocks?

Moreover, have you tried generating the `volume` baseline using `orca`?

Reply:
I think there is one mock that caused a problem. Now with @antoinerg parallel PR in place I could fix this. 
Second question: No I have not tried `volume` with `orca`.

Reply:
> Second question: No I have not tried `volume` with `orca`.

Cool, could you give it a try?

Reply:
See

https://github.com/plotly/plotly.js/blob/c6573487f33ffe02f38f443690e5fdb824fdb161/tasks/noci_test.sh#L31-L38

for a sample orca command.